### ISSUE TITLE
Bug on with Polymer 2 (key events aren't catched)

### ISSUE THREAD

**Issue by RoXuS** [PolymerElements/iron-a11y-keys-behavior#78]
### Description

If I don't add the following lines in the `ready()` function, the key events are not catched:

```javascript
    ready() {
      super.ready();
      this._prepKeyBindings();
      this.keyEventTarget = document.body;
    }
```

### Live Demo
http://jsbin.com/zeraputiyi/1/edit?html,console,output

### Browsers Affected
- [x] Chrome

**Comment by valdrinkoshi** [comment]
`keyBindings` is expected to be a prototype property - [see demo as example](https://github.com/PolymerElements/iron-a11y-keys-behavior/blob/master/demo/x-key-aware.html). `IronA11yKeysBehavior` will collect the key bindings when registered, but the properties from the properties block will be created when the element is connected (later).

tl;dr you can have a getter for `keyBindings` to solve this issue http://jsbin.com/dazedes/1/edit?html,console,output

**Comment by RoXuS** [comment]
Oh great ! Thx !
